package.json

```diff
@@ -221,14 +218,13 @@
         "@types/pixelmatch": "^5.2.4",
         "@types/pngjs": "^6.0.1",
         "@types/query-selector-shadow-dom": "^1.0.0",
-        "@types/react": "^16.14.2",
-        "@types/react-dom": "^16.9.8",
+        "@types/react": "^17.0.39",
+        "@types/react-dom": "^18.2.8",
```

Comment:
These should have `^18.2.0` just like their respective packages

Reply:
It originally was but caused a ton of type issues, caused by this issue I believe: https://github.com/facebook/react/issues/24304.

As far as I could tell a lot of our antd components were affected. I tried upgrading antd to confirm but that was a task in and of itself.

Is there another way around this? Or any reason it can't stay at v17 temporarily